### ISSUE TITLE
Follow up changes after PR #7782 is merged

### ISSUE THREAD

**Issue by Patowhiz** [africanmathsinitiative/R-Instat#7852]
@rdstern @lloyddewit @africanmathsinitiative/developers 
PR #7782 makes several underlying changes that require some dialogs implementations to be changed.
The following are the dialogs that will have to be modified after the PR is merged.

**Hight priority**
1. View object and View Graph dialog - @rdstern I propose the 2 dialogs to now be merged into one. We now have the underlying R and VB.Net implementation that allows this be easily done. Then also the concept of display options can now be changed. I assign myself this. We could have 1 dialog called view objects, capable of viewing; graphs, tables, models, structures and summaries.
2. Renaming, reordering and deleting of objects -  these dialogs and the underlying R code also needs to be refactored.
3. All dialogs that produce tables - they now have to supply the `object_format` depending on the table format. I suggest @EstherNjeriLiberatta  to do this.
4. All dialogs that produce models - they now have to supply the `object_format`. I suggest @Vitalis95 to do this.
5. All dialogs that produce text summaries - they can now specify the `object_format` this in their SetAssign. If @rdstern thinks it's a great idea to introduce the discussed aspects of saving these summaries as objects too, the dialogs can now have a save control. I suggest @anastasia-mbithe to do this. This promotes reproducibility aspects that we have with other objects
6. All dialogs that produce survival objects - From our discussions with @lilyclements , these are now structured objects which are in text formats. So the save control should adopt the 2 specifications.

**Low priority**
1. All dialogs that produce a graph. They just need to be amended to explicitly pass the `object_format`. This will make them consistent with the other dialogs. I suggest John to do this. 

Once the above is all done, we can simplify and optimise our R link and output implementations even more.
Note the above can be done once PR#7782 is merged.

**Comment by Patowhiz** [comment]
In addition, as mention in PR#7782, **Describe > Two/Three variables > Summarise** dialog needs discussions with @rdstern on how we can produce 1 table at a time. Currently it produces 2 html tables in 1 command. I suggest @N-thony to look at this. He may wish to delegate too.

**Comment by N-thony** [comment]
@Patowhiz as now PR #7853, we can have a quick discussion about everything above and then I will enable the team how can do it systematically.

**Comment by N-thony** [comment]
> @Patowhiz as now PR #7853, we can have a quick discussion about everything above and then I will enable the team how can do it systematically.

@Patowhiz could we have this chat today? Let me know when you are available.

**Comment by dannyparsons** [comment]
@Patowhiz Have you confirmed that 9. is working correctly - backward compatibility of older files?

**Comment by Patowhiz** [comment]
@dannyparsons not yet. I haven't added the support.
@rdstern changed my priorities to;
1. Model dialogs - this one expanded the requirements to support multiple scripts that produce different outputs (@dannyparsons we will have to discuss a concrete system for these kind of dialog scripts). 
2. Two and three variable frequencies dialogs - this one expanded the requirements to handle grid outputs (I'm about to write an issue about this).
3. Other dialogs that have html outputs.
4. Base graphics as discussed with @dannyparsons.
